openreview/conference/templates/areachairWebfield.js

```diff
@@ -694,11 +694,12 @@ var renderStatusTable = function(conferenceStatusData, container) {
       '</ul>' +
     '</div>' +
     '<div class="btn-group"><button class="btn btn-export-data" type="button">Export</button></div>' +
+    '<div class="btn-group"><button class="btn btn-export-pdf" type="button">Download PDF</button></div>' +
```

Comment:
Should this say "Download PDFs" or "Download all PDFs"?

Reply:
the label is shorten so that it can fit in the row without taking too much space from search and sort